sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/Models/AuthorizationProviderOperationInfo.Serialization.cs

```diff
@@ -49,15 +49,22 @@ void IJsonModel<AuthorizationProviderOperationInfo>.Write(Utf8JsonWriter writer,
             }
             if (Optional.IsDefined(Properties))
             {
-                writer.WritePropertyName("properties"u8);
+                if (Properties != null)
```

Comment:
Modeler4 emit `Properties` is nullable. so this change is expected

Reply:
The existing implementation in feature/v3 is missing nullability for `UnknownType` [here](https://github.com/Azure/autorest.csharp/blob/c875f2eac3eaa0995ffdb0bb8f1ab77dd4d9719e/src/AutoRest.CSharp/Common/Generation/Types/TypeFactory.cs#L83)
This was introduced in https://github.com/Azure/autorest.csharp/pull/4707/files#diff-1e9b6e9b8a74a8508034ac58f3b0fc0cf9fd9a1ba37cec4e4b2d4a85b1e90c4fR83

All the other changes are the same case, where `BinaryData` is the property type. This PR fixes them all.